packages/reaction-core/common/methods/cart.js

```diff
@@ -4,8 +4,9 @@
 //
 if (Meteor.isClient) {
   Meteor.methods({
-    "cart/submitPayment": function (paymentMethod) {
+    "cart/submitPayment": function (paymentMethod, sessionId) {
```

Comment:
agree, but we could leave it like this due to identically of all `cart/submitPayment`. Or should I update it?